plugins/catalog-backend/src/database/CommonDatabase.ts

```diff
@@ -319,6 +319,21 @@ export class CommonDatabase implements Database {
     return toEntityResponse(rows[0]);
   }
 
+  async entityById(
```

Comment:
`entityByUid` to match the names in all the other places

Reply:
Done